src/test/java/org/elasticsearch/action/IndicesRequestTests.java

```diff
@@ -167,19 +177,45 @@ public void testGetFieldMappings() {
         assertSameIndices(getFieldMappingsRequest, getFieldMappingsShardAction);
     }
 
+
     @Test
-    public void testAnalyze() {
+    public void testAnalyze() throws IOException {
         String analyzeShardAction = AnalyzeAction.NAME + "[s]";
         interceptTransportActions(analyzeShardAction);
 
         AnalyzeRequest analyzeRequest = new AnalyzeRequest(randomIndexOrAlias());
         analyzeRequest.text("text");
-        internalCluster().clientNodeClient().admin().indices().analyze(analyzeRequest).actionGet();
+        AnalyzeResponse response =  internalCluster().clientNodeClient().admin().indices().analyze(analyzeRequest).actionGet();
+        AnalyzeResponse.AnalyzeToken token = response.getTokens().get(0);
+        assertFalse(token.isAlldata());
+        XContentBuilder builder = JsonXContent.contentBuilder();
+        response.toXContent(builder,null);
+        String str = new String(builder.bytes().array(), Charset.forName("UTF8"));
+        assertFalse(str.contains("\"flags\":"));
+        clearInterceptedActions();
+        assertSameIndices(analyzeRequest, analyzeShardAction);
+    }
```

Comment:
Think this test is more usefull rather when before. Now it has some assertions about response while test before was just about "no errors"